examples/chip-tool/commands/tests/Commands.h

```diff
@@ -7829,6 +7832,75 @@ class TestCluster : public TestCommand
 
         runner->NextTest();
     }
+
+    // Test Send Test Command to unsupported endpoint
+    typedef void (*SuccessCallback_101)(void * context);
```

Comment:
nit: Prefer `using`as it is more readable than `typedef`

Reply:
Again, this is generated code.  Happy to look into this in a followup, but don't think we should be including changes to the generator templates in this PR.